### ISSUE TITLE
Password field (ipywidgets.Password) is not properly styled

### ISSUE THREAD

**Issue by earshinov** [jupyter-widgets/ipywidgets#2492]
**Describe the bug**

Password field (`ipywidgets.Password`) is not properly styled.

Here it is in comparison with a simple text field (`ipywidgets.Text`):

![text vs password field](https://user-images.githubusercontent.com/52021/60767635-48165200-a0c3-11e9-980c-9a11892f82f5.png)

A bunch of styles is defined for `.widget-text input[type="text"], .widget-text input[type="number"], .widget-textarea textarea`, but as you can see, `input[type="password"]` is not in the list.

**To reproduce**

In a Jupyter notebook:

```python
import ipywidgets as widgets

display(widgets.VBox([
  widgets.Text(description='Username')
  widgets.Password(description='Password')
]))
```

**Expected behavior**

Password fields should have the same size, paddings, border color etc. as ordinary text fields.

**Environment:**

- ipywidgets  7.5.0

Original issue filed to JupyterLab: https://github.com/jupyterlab/jupyterlab/issues/6777

**Comment by SylvainCorlay** [comment]
Thanks @earshinov !

This is certainly a bug. I have marked it as a "first good issue" for anyone interested in fixing it.

**Comment by jasongrout** [comment]
Fixed by #2619.
